src/dune_tui/drawing/user_message_to_image.ml

```diff
@@ -0,0 +1,111 @@
+open Stdune
+
+let attr_of_ansi_color_rgb8 (c : Ansi_color.RGB8.t) =
+  let module A = Notty.A in
+  match Ansi_color.RGB8.to_int c with
+  | 0 -> A.black
+  | 1 -> A.red
+  | 2 -> A.green
+  | 3 -> A.yellow
+  | 4 -> A.blue
+  | 5 -> A.magenta
+  | 6 -> A.cyan
+  | 7 -> A.white
+  | 8 -> A.lightblack
+  | 9 -> A.lightred
+  | 10 -> A.lightgreen
+  | 11 -> A.lightyellow
+  | 12 -> A.lightblue
+  | 13 -> A.lightmagenta
+  | 14 -> A.lightcyan
+  | 15 -> A.lightwhite
+  | i when i <= 231 ->
+    let i = i - 16 in
+    let r = i / 36 in
+    let g = i / 6 mod 6 in
+    let b = i mod 6 in
+    A.rgb ~r ~g ~b
+  | i when i <= 255 -> A.gray (i - 232)
+  | i -> Code_error.raise "invalid 8-bit color" [ "value", Dyn.int i ]
```

Comment:
This isn't new, but this behavior is surprising. We shouldn't crash with a Code_error on bad input. We should probably just ignore.

Reply:
However it is imossible for this Code_error to raise. It is matching on the output of Ansi_color.RGB8.to_int which can only be < 256.